core/src/main/java/nl/nn/adapterframework/align/Json2Xml.java

```diff
@@ -336,6 +334,53 @@ protected void processChildElement(JsonValue node, String parentName, XSElementD
 		super.processChildElement(node, parentName, childElementDeclaration, mandatory, processedChildren);
 	}
 
+	@Override
```

Comment:
Ik maak hier niet echt uit op _waarom_ je deze twee methods override. kan je toevoegen waarom je dit doet?

Reply:
Omdat afhankelijk van het type je wilt weten of hij empty is?

Reply:
Ik heb de base methods nu abstract gemaakt, zodat dat duidelijker is. En dummy-implementations in de DomAligner / Properties2Xml.

Reply:
Yes, helder